src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -119,6 +121,8 @@ public JsonSchemaTransformer(MarkdownEngine markdownEngine, LinkResolver linkRes
                     LazyThreadSafetyMode.PublicationOnly);
             }
 
+            xref.Monikers = monikerlist;
```

Comment:
I think it would be cleaner to invlove `MonikerProvider` as dependency to `JsonSchemaTransformer` dierctly, no need to pass along `MonikerList` all the way